beacon-chain/core/blocks/block_operations.go

```diff
@@ -445,10 +447,10 @@ func VerifyAttesterSlashing(beaconState *pb.BeaconState, slashing *ethpb.Atteste
 	if !IsSlashableAttestationData(data1, data2) {
 		return errors.New("attestations are not slashable")
 	}
-	if err := VerifyIndexedAttestation(beaconState, att1); err != nil {
+	if err := VerifyIndexedAttestation(context.TODO(), beaconState, att1); err != nil {
```

Comment:
Is `Todo()` ok here? Since we don't care about tracing for `VerifyAttesterSlashing`

Reply:
Talked with @prestonvanloon, will follow best practice and pass a real ctx here